src/integrationTest/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersControllerIT.java

```diff
@@ -227,6 +229,7 @@ private void verifyDbCpoAuditValues(UUID id,
             assertEquals(CREATED_BY_IDAM_MOCK, latestRevision.getEntity().getCreatedBy());
             assertNotNull(latestRevision.getEntity().getCreatedTimestamp());
             assertTrue(beforeCreateTimestamp.isBefore(latestRevision.getEntity().getCreatedTimestamp()));
+            assertFalse(latestRevision.getEntity().isHistoryExists());
```

Comment:
Change to `.andExpect(jsonPath("$.history_exists", is(HISTORY_EXISTS_UPDATED)))` as we know it should change for an Update.

Reply:
Sorry cut-n-paste error on my part:  try `assertEquals(HISTORY_EXISTS_DEFAULT, savedEntity.get().isHistoryExists());`  rather than assertFalse: to highlight that we expect it to default.